actions/commit-and-push/action.yaml

```diff
@@ -18,17 +18,29 @@ inputs:
     description: "The ref name to commit and push the files on."
     required: false
     default: ${{ github.event.repository.default_branch }}
+  add-untracked:
+    description: "Whether to add untracked files to commit."
+    required: false
+    default: "false"
 
 runs:
   using: "composite"
   steps:
     # For more information to create a commit with a valid GitHub user go to:
     # https://github.com/ad-m/github-push-action
-    - name: Commit all files
+    - name: Configure git
       run: |
         git config user.email ${{ inputs.github-email }}
         git config user.name ${{ inputs.github-username }}
-        git commit -m "${{ inputs.commit-message }}" -a
+      shell: bash
+
+    - name: Add untracked files
+      run: git add .
+      if: ${{ inputs.add-untracked == 'true' }}
+      shell: bash
+
+    - name: Commit files
+      run: git commit -m "${{ inputs.commit-message }}" -a
```

Comment:
```suggestion
      run: git commit -m "${{ inputs.commit-message }}"
```

Reply:
I wouldn't like to change this as it's been the default before.
This makes sure that already tracked files are included in the commit. To include all changed files (untracked ones) I added a separate setting.